api/db/changelog/db.changelog-12.xml

```diff
@@ -7,43 +7,76 @@
         http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
     <changeSet author="brianfreeman" id="changelog-12">
 
-        <createTable tableName="participant_cohort_status">
-            <column name="cohort_id" type="BIGINT">
-                <constraints nullable="false"/>
+        <createTable tableName="cohort_review">
+            <column name="cohort_review_id" type="bigint" autoIncrement="true">
+                <constraints primaryKey="true" nullable="false"/>
             </column>
-            <column name="cdr_version_id" type="BIGINT">
+            <column name="cohort_id" type="bigint">
                 <constraints nullable="false"/>
             </column>
-            <column name="participant_id" type="BIGINT">
+            <column name="cdr_version_id" type="bigint">
                 <constraints nullable="false"/>
             </column>
-            <column name="status" type="tinyint">
+            <column name="creation_time" type="datetime">
```

Comment:
It seems like there needs to be a status field here of some kind... how do we know whether the user has ever actually initiated a review in this representation?